converter/config/mapping_ae-usi_to_datamodel.json

```diff
@@ -246,16 +342,10 @@
           "path": ["protocolRefs"],
           "from_type": "array",
           "method": "get_reference_value_from_json"
-        }
-      }
-    },
-    "projectref": {
```

Comment:
Projectref is needed for USI. Can you please put it back?

Reply:
Do you need Projectrefs too?

Reply:
Where do you see "projectrefs"? I think there should be only one "projectref" (in the study object).

Reply:
My mistake. I added this back.